examples/cms-sitefinity/pages/_document.tsx

```diff
@@ -0,0 +1,15 @@
+import Document, { Html, Head, Main, NextScript } from 'next/document'
```

Comment:
This does not do anything, can we just remove it?

Reply:
I get errors without the import:

![image](https://user-images.githubusercontent.com/4221529/193187886-042b95af-055c-4b0e-891a-cad0fdf2604a.png)

Reply:
I meant the whole file.

Reply:
Removed it. :+1: